lib/workers/pr/body/__snapshots__/index.spec.ts.snap

```diff
@@ -0,0 +1,14 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`workers/pr/body generate the message markdown 1`] = `
+"This PR contains the following updates:
+
+| Package | Type | Update | Change |
+|---|---|---|---|
+| [@&#8203;scope/package-with-link](http://scope.com/package-with-link) | dependencies | patch | \`^10.1.1\` -> \`^10.1.2\` |
+| @&#8203;scope/package-without-link | dependencies | patch | \`^10.1.1\` -> \`^10.1.2\` |
```

Comment:
should we quote this as `@&#8203;scope/package-without-link`

Reply:
Maybe in a separate PR